flutter_dribbble_nike_sb/lib/widgets/app_cta_button.dart

```diff
@@ -22,7 +22,7 @@ class AppCtaButton extends StatefulWidget {
 
 class _AppCtaButtonState extends State<AppCtaButton> {
   String get _title => widget.title;
-  Function get _onCtaTap => widget.onCtaTap;
+  Function/*!*/ get _onCtaTap => widget.onCtaTap;
```

Comment:
Similar to this one, kindly check the rest. Here's a reference that can help you: https://dart.dev/null-safety/migration-guide